api/Makefile

```diff
@@ -3,7 +3,7 @@ dev:
 	CONFIG_YAML_DIRECTORY=./ TZ=utc APP_ENV=development go run main.go
 
 test:
-	CONFIG_YAML_DIRECTORY=../.. TZ=utc APP_ENV=test go test -v ./... -run ^$(name)
+	CONFIG_YAML_DIRECTORY=../.. TZ=utc APP_ENV=test go test ./... -run ^$(name)
```

Comment:
removed this -v because it was really difficult to read which tests were failing